src/js/_store/modules/categories.js

```diff
@@ -3,16 +3,48 @@ import axios from '@Helpers/axiosDefault';
 export default {
   namespaced: true,
   state: {
+    targetCategories: {
```

Comment:
[imo]

変数名やプロパティ名を複数形にするときは基本的には、配列の時にしたほうが良いかなと思います！

```suggestion
    targetCategory: {
```

Reply:
承知しました！複数のカテゴリーについての記述なので、複数形にする
という認識でいたのですが、今後は配列の時のみ複数形で記述を行うようにいたします